### PR TITLE
Fullscreen / auto refresh [WIP]

### PR DESCRIPTION
**Pull request by tlrobinson**

### CODE REVIEW

File: frontend/src/dashboard/components/RefreshWidget.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+];
+
+export default class RefreshWidget extends Component {
+    render() {
+        const { period, elapsed, onChangePeriod } = this.props;
+        const remaining = period - elapsed;
+        return (
+            <PopoverWithTrigger
+                ref="popover"
+                triggerElement={elapsed == null ?
```
Comment (pygeek) [inline frontend/src/dashboard/components/RefreshWidget.jsx]: Is there a cleaner way to do this in React.js?

Comment (pygeek) [inline frontend/src/dashboard/components/RefreshWidget.jsx]: I'm just thinking that this would be difficult to  test.

**Comment by agilliland** [comment]
@tlrobinson are we closing this in favor of #2136 ?

**Comment by tlrobinson** [comment]
Yup

**Pull request closed without merging**